frontend/components/Timer.js

```diff
@@ -0,0 +1,86 @@
+import React, {useEffect, useState} from 'react';
+import {Box, Button, Center, HStack, StatusBar, Text, View} from "native-base";
+import { FontAwesome } from '@expo/vector-icons';
+import {StyleSheet} from "react-native";
+import Api from "./util/Api";
+
+const Timer = () => {
+    const bgColor = "tertiary.800"
+    const [seconds, setSeconds] = useState(0)
+    const [minutes, setMinutes] = useState(0)
+    const [isActive, setIsActive] = useState(false)
+
+    const toggle = () => {
+        setIsActive(!isActive)
+    }
+
+    const reset = () => {
+        sendFinalTimeToDatabase()
+        setMinutes(0)
+        setSeconds(0)
+        setIsActive(false)
+    }
+
+    const sendTimeToDatabase = () => {
+        Api.post('/timer/token', {
+            minutes: minutes,
+            seconds: seconds
+        }).then(res => {
+            console.log(res)
+        }).catch(e => {
+            console.log(e)
+        })
+    }
+
+    const sendFinalTimeToDatabase = () => {
+        Api.post('/timer-end/token', {
+            minutes: minutes,
+            seconds: seconds
+        }).then(res => {
+            console.log(res)
+        }).catch(e => {
+            console.log(e)
+        })
+    }
```

Comment:
Duplicate code, maybe we should consider passing path param to function above if it's intuitive enough.